manifests/2.8.0/opensearch-2.8.0.yml

```diff
@@ -91,3 +91,11 @@ components:
     checks:
       - gradle:properties:version
       - gradle:dependencies:opensearch.version
+  - name: index-management
+    repository: https://github.com/opensearch-project/index-management.git
+    ref: tags/2.8.0.0
```

Comment:
Hi @xuezhou25, this tag version for repo wouldn't be created until we launched the release. I think in you case this should be referring to 2.8 branch.